src/code/selectedOp.js

```diff
@@ -192,25 +192,23 @@ export async function setOpBackground(opID, background) {
 
 export async function opsList(hidden = true) {
   // after 0.19, remove the list and just query the idb keys
```

Comment:
per this comment, do we still need to maintain the list?

Reply:
that's only to catch pre 0.19 local ops